web-server/src/comment/comment.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Post } from '../post/post.entity';
+import {
+    Column,
+    CreateDateColumn,
+    Entity,
+    PrimaryGeneratedColumn,
+    Tree,
+    ManyToOne,
+    TreeChildren,
+    TreeParent
+} from 'typeorm';
+
+@Entity()
+@Tree("closure-table")
+export class Comment {
+    @PrimaryGeneratedColumn()
+    readonly id: number;
+
+    @CreateDateColumn()
```

Comment:
date column might be a trouble in tests(since it uses the dates from database implementations) -> any suggestions as to how we can improve it?

Reply:
We could generate the date in the backend and then save it along with the other data.

Reply:
exactly what we are doing in our mono repo -> we made a date service that can be mocked out in testcases.